stdpopsim/models.py

```diff
@@ -278,7 +278,7 @@ def _get_C_from_N(self, N):
         C = np.zeros([n**2, n**2])
         index_array = np.array(range(n**2)).reshape([n, n])
         for idx in range(n):
-            C[index_array[idx, idx], index_array[idx, idx]] = 1 / (2 * N[idx])
```

Comment:
Huh, this is troubling. There *should* be a factor of 2 there. I think this means we're missing a factor of 2 somewhere else.